src/constants/index.ts

```diff
@@ -2,6 +2,43 @@ import { constants, RpcProvider } from "starknet";
 
 export const STRK_TOKEN =
   "0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d" as const;
+
+export interface Token {
+  symbol: string;
+  address: string;
+  logoUrl: string;
+  decimals: number;
+  balance?: number;
+}
+
+export const TOKENS: Token[] = [
+  { symbol: 'ETH', address: '0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7', logoUrl: 'https://assets.coingecko.com/coins/images/279/small/ethereum.png', decimals: 18 },
```

Comment:
use the logo's from here

[strkfarm/starkfarm-client@c8b2b74/src/constants.ts#L6](https://github.com/strkfarm/starkfarm-client/blob/c8b2b745e8b15ec94102218df2145cb4002a467a/src/constants.ts#L6)
xSTRK logo: http://endur.fi/logo.svg